### PR TITLE
Fixed Issue #32

### PR DESCRIPTION
**Pull request by Mayank1403**
Event heading will take you to corresponding event page. I have added comingSoon.js inside pages.

### CODE REVIEW

File: pages/events.js [hunk 1]
```diff
@@ -1,97 +1,128 @@
...
+              src={`${process.env.ASSET_PREFIX}/images/mlab/hackathon.png`}
+            />
+          </div>
+          <div className="eventSectionText">
+            <Link
+              className="eventPageLink"
+              href={`${process.env.ASSET_PREFIX}/hashcode`}
+            >
+              <Typography className="eventHeading">HashCode</Typography>
+            </Link>
```
Comment (DarkAEther) [inline pages/events.js:32]: When the page is loaded, it is not immediately obvious that the Event Heading is a link to the page. I suggest maybe an arrow or some kind of hover effect that would make it easy for the visitor to implicitly understand that it is a link that they can click on for more information.

This applies to the other events as well.

Comment (Mayank1403) [reply pages/events.js:32]: Okay, I will add on hover effect on it.

**Comment by DarkAEther** [comment]
Looks good to me! Once @souravtecken takes a look, we can go ahead and merge this.

**Review by souravtecken — APPROVED** [review]
Looks good to me, @Mayank1403, love the little hover effect you've added!

**Comment by souravtecken** [comment]
On a side note, @Mayank1403 @DarkAEther, what do you guys think of adding that same animation to all links on the site? I guess it'll look cool? xD

**Comment by Mayank1403** [comment]
Thank you for compliments @souravtecken 
@DarkAEther I think that will be good. It's worth giving a try

**Pull request merged**